### PR TITLE
Added section on sync and aliases in dashboards FAQ

### PR DESCRIPTION
**Pull request by goransle**

### CODE REVIEW

File: docs/dashboards/frequently-asked-questions.md [hunk 1]
```diff
@@ -121,3 +121,41 @@ The layout of our library is based on flexbox, so in general all browsers, which
...
+            // set an alias that matches the series name
+            'customName': 'water',
+            'otherCustomName': 'air'
+          }
+        }
+      }
+    }]
+  }
+```
+See [jsfiddle.net/goransle/udcfyL8b/](https://jsfiddle.net/goransle/udcfyL8b/) for a working examples
```
Comment (karolkolodziej) [inline docs/dashboards/frequently-asked-questions.md:159]: Why don't we add this to our demos?

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

File: docs/dashboards/frequently-asked-questions.md (hunk 1 above)
Comment (goransle) [reply docs/dashboards/frequently-asked-questions.md:159]: This should ideally be a temporary workaround, so not sure if it's worth bloating our samples with it.

Comment (karolkolodziej) [reply docs/dashboards/frequently-asked-questions.md:159]: Sure, please than improve the fiddle so everyone can reach it:
I'm talking about `http://localhost:3031/`

**Review by bre1470 — APPROVED** [review]

File: docs/dashboards/frequently-asked-questions.md (hunk 1 above)
Comment (goransle) [reply docs/dashboards/frequently-asked-questions.md]: Looks like github.highcharts.com is broken for dashboards now, so might as well make a sample 😓

**Review by sebastianbochan — APPROVED** [review]

**Review by karolkolodziej — APPROVED** [review]

**Pull request merged**